build/sections/qualitat.html

```diff
@@ -0,0 +1,34 @@
+<div id="qualitat" class="bg-[url('../Images/qualitat.png')] bg-cover pt-60 pb-28 text-white container relative">
+
+<div class="mr-6 pb-10 lg:mr-[35rem] ">
+    <p class="font-raleway font-bold  text-2xl pb-5 ">
+        Qualität steht bei uns an erster Stelle. Mit unserem <span class=" text-primary-green">Trainingsprogramm</span> direkt Vorort werden die juniorigen Mitarbeitenden auf das projektgeschäft vorbereitet.
+    </p>
+    <p class="font-montserrat font-medium text-base">
+        kurze Einführung und den Bezug zu MW darstellen At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est L vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est L
+    </p>
+</div> 
+<div class="flex flex-col gap-20 lg:flex-row justify-center lg:gap-28 pb-11">
+    <div class=" flex justify-between border-b-2 border-primary-green w-[19.5rem] lg:w-[12.5rem] ">
+        <label>
+            <input value="Coding skills" class="bg-transparent flex-1">
+        </label>
+        <label>+</label>
+    </div>
+    <div class="border-b-2 border-primary-green w-[19.3rem]">
+        <label>
+            <input value="Kommunikations- und Teamfähigkeit" class=" bg-transparent  w-72 ">
+        </label>
+        <label>+</label>
+    </div>
+</div>
+<div>
+    <div class="py-12 lg:mr-[28rem]">
+        Mehr dazu von Matthias Moosburger, Senior Lead IT Architect bei MaibornWolff, 
+        der das Ausbildungsprogramm maßgeblich entwickelt hat:
```

Comment:
Optional: Increase font-size and use p tag instead of a div

Reply:
I have done that 👍🏾